packages/gestalt/src/PageHeader.js

```diff
@@ -27,9 +28,11 @@ export type ActionType = Element<
 
 type Props = {|
   /**
-   * Add [Badge](https://gestalt.pinterest.systems/web/badge) displayed after the title. Be sure to localize the text. See the [title variant](https://gestalt.pinterest.systems/web/pageheader#Title) to learn more.
+   * Adds [Badge](https://gestalt.pinterest.systems/web/badge) displayed after the title. Be sure to localize the text. See the [title variant](https://gestalt.pinterest.systems/web/pageheader#Title) to learn more.
+   *
+   * \`type\` option determines the style of the badge. The default value is "info". See the [type](https://gestalt.pinterest.systems/web/badge#Type) variant to learn more.
    */
-  badge?: {| text: string, tooltipText?: string |},
+  badge?: {| text: string, tooltipText?: string, type?: TypeOptions |},
```

Comment:
we don't export/import types (yet) to prevent this: 
![Screenshot by Dropbox Capture](https://github.com/pinterest/gestalt/assets/10593890/5fafbee1-fa18-42c9-915a-3da7a4aced73)

Ravi has ideas on how to fix this. We've been discussing it. 

For now, hard typed... :(

Reply:
That was my thought too :D. Will update it 👍 .